woocommerce/payment-gateway/Frontend/Script_Handler.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * WooCommerce Payment Gateway Framework
+ *
+ * This source file is subject to the GNU General Public License v3.0
+ * that is bundled with this package in the file license.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://www.gnu.org/licenses/gpl-3.0.html
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@skyverge.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade the plugin to newer
+ * versions in the future. If you wish to customize the plugin for your
+ * needs please refer to http://www.skyverge.com
+ *
+ * @package   SkyVerge/WooCommerce/Payment-Gateway/Classes
+ * @author    SkyVerge
+ * @copyright Copyright (c) 2013-2020, SkyVerge, Inc.
+ * @license   http://www.gnu.org/licenses/gpl-3.0.html GNU General Public License v3.0
+ */
+
+namespace SkyVerge\WooCommerce\PluginFramework\v5_6_1\Frontend;
+
+defined( 'ABSPATH' ) or exit;
+
+if ( ! class_exists( '\\SkyVerge\\WooCommerce\\PluginFramework\\v5_6_1\\Frontend\\Script_Handler' ) ) :
+
+
+/**
+ * Script Handler Abstract Class
+ *
+ * Handles initializing the payment registered JavaScripts
+ *
+ * @since x.y.z
+ */
+abstract class Script_Handler {
+
+
+	/** @var string JS handler base class name, without the FW version */
+	protected $js_handler_base_class_name = '';
+
+
+	/**
+	 * Returns the JS handler class name.
+	 *
+	 * @since x.y.z
+	 *
+	 * @return string
+	 */
+	protected function get_js_handler_class_name() {
+
+		return sprintf( '%s_5_6_1', $this->js_handler_base_class_name );
```

Comment:
```suggestion
		return sprintf( '%s_v5_6_1', $this->js_handler_base_class_name );
```

hey guys @ChaseWiseman @wvega @deivamagalhaes just had an afterthought about the inline JS namespace containing the script version adapted from the framework semver. Should we prepend a `v` here? So it matches the namespace in PHP classes e.g. `SkyVerge\WooCommerce\PluginFramework\v5_6_1\Frontend`; concern is just if we mass rename these what if we are doing search & replace with `v5_6_1` rather than `5_6_1` as the search needle?

Reply:
Adjusted in https://github.com/skyverge/wc-plugin-framework/pull/475